### PR TITLE
Update return block documentation for rds modules

### PR DESCRIPTION
**Pull request by GomathiselviS**
##### SUMMARY

Refer: https://issues.redhat.com/browse/ACA-1484

This PR updates the return block in the rds module documentation.

##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: plugins/modules/rds_snapshot_info.py [hunk 1]
```diff
@@ -69,116 +69,6 @@
...
-      returned: when snapshot is not shared
-      type: complex
-      contains: {}
-    vpc_id:
-      description: ID of VPC containing the DB
-      returned: always
-      type: str
-      sample: vpc-abcd1234
 cluster_snapshots:
   description: List of cluster snapshots
```
Comment (alinabuzachis) [inline plugins/modules/rds_snapshot_info.py:73]:
```suggestion
  description: List of cluster snapshots.
```

File: plugins/modules/rds_snapshot_info.py [hunk 2]
```diff
@@ -200,32 +90,32 @@
       description: Date and time the cluster was created
```
Comment (alinabuzachis) [inline plugins/modules/rds_snapshot_info.py:90]:
```suggestion
      description: Date and time the cluster was created.
```

File: plugins/modules/rds_snapshot_info.py [hunk 3]
```diff
@@ -200,32 +90,32 @@
...
       returned: always
       type: str
-      sample: '2018-05-17T00:13:40.223000+00:00'
+      sample: "2018-05-17T00:13:40.223000+00:00"
     db_cluster_identifier:
       description: Database cluster identifier
```
Comment (alinabuzachis) [inline plugins/modules/rds_snapshot_info.py:95]:
```suggestion
      description: Database cluster identifier.
```

File: plugins/modules/rds_snapshot_info.py [hunk 4]
```diff
@@ -200,32 +90,32 @@
...
       returned: always
       type: str
-      sample: test-aurora-cluster
+      sample: "test-aurora-cluster"
     db_cluster_snapshot_arn:
       description: ARN of the database snapshot
```
Comment (alinabuzachis) [inline plugins/modules/rds_snapshot_info.py:100]:
```suggestion
      description: ARN of the database snapshot.
```

File: plugins/modules/rds_snapshot_info.py [hunk 5]
```diff
@@ -200,32 +90,32 @@
...
       returned: always
       type: str
-      sample: arn:aws:rds:ca-central-1:123456789012:cluster-snapshot:test-aurora-snapshot
+      sample: "arn:aws:rds:ca-central-1:123456789012:cluster-snapshot:test-aurora-snapshot"
     db_cluster_snapshot_identifier:
       description: Snapshot identifier
```
Comment (alinabuzachis) [inline plugins/modules/rds_snapshot_info.py:105]:
```suggestion
      description: Snapshot identifier.
```

File: plugins/modules/rds_snapshot_info.py [hunk 6]
```diff
@@ -200,32 +90,32 @@
...
       returned: always
       type: str
-      sample: test-aurora-snapshot
+      sample: "test-aurora-snapshot"
     engine:
       description: Database engine
```
Comment (alinabuzachis) [inline plugins/modules/rds_snapshot_info.py:110]:
```suggestion
      description: Database engine.
```

**Review by alinabuzachis — COMMENTED** [review]
Can you also please update description of returned values to end with a period. Thanks.

**Review by gravesm — APPROVED** [review]

**Review by hakbailey — APPROVED** [review]

**Review by alinabuzachis — APPROVED** [review]

**Pull request merged**